src/mozanalysis/experiment.py

```diff
@@ -173,16 +173,29 @@ def get_single_window_data(
             analysis_length_days, self.num_dates_enrollment
         )
 
-        sql = self.build_query(
-            metric_list, time_limits, enrollments_query_type, custom_enrollments_query,
-            segment_list
+        sql_to_be_hashed = self.build_query(
+            metric_list=metric_list,
+            time_limits=time_limits,
+            destination_table="",
```

Comment:
I'm kind of disappointed that you didn't go for a crypto-mining like solution, where you need to keep trying different values here until you find one that matches the hash.